sci-calculators/bc-gh/bc-gh-2.3.1.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Implementation of POSIX bc with GNU extensions"
+HOMEPAGE="https://github.com/gavinhoward/bc"
+SRC_URI="https://github.com/gavinhoward/bc/releases/download/${PV}/bc-${PV}.tar.xz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~ia64 ~ppc ~ppc64 ~s390 ~sh ~x86"
```

Comment:
You can only add ~amd64 or ~x86 for new packages, and even ~x86 requires you to test it on x86 architechture. For the rest, you need to belong to these arch teams before you can add the keywords.

Reply:
Fixed.